teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'unknown')
```

Comment:
Please ensure that you add `parent_permission = true` to the `initialize params` according to the project requirement.

Reply:
Alright I will do it.